setup-machine-arktos.sh

```diff
@@ -146,6 +142,26 @@ sudo pip3 install netaddr
 sudo pip3 install kopf
 sudo pip3 install PyYAML
 
+#####################
+
+echo Setup: Install Containerd
+
+cd $HOME
+wget https://github.com/containerd/containerd/releases/download/v1.4.2/containerd-1.4.2-linux-amd64.tar.gz
+cd /usr
+sudo tar xvf $HOME/containerd-1.4.2-linux-amd64.tar.gz
+
+cd $HOME/go/src/k8s.io/arktos
+wget -qO- https://github.com/futurewei-cloud/containerd/releases/download/tenant-cni-args/containerd.zip | zcat > /tmp/containerd
```

Comment:
We should not use futurewei-cloud, use CentaurusInfra instead.

Reply:
I have replaced the occurrences of futurewei-cloud with CentaurusInfra in setup-machine-arktos.sh and replace-containerd.sh files